packages/components/src/popover/test/index.tsx

```diff
@@ -129,32 +125,43 @@ describe( 'Popover', () => {
 					);
 				};
 
-				await renderAsync(
+				render(
 					<PopoverWithAnchor>Popover content</PopoverWithAnchor>
 				);
 
-				expect(
-					screen.getByText( 'Popover content' )
-				).toBeInTheDocument();
+				await waitFor( () =>
+					expect(
+						screen.getByText( 'Popover content' )
+					).toBeVisible()
+				);
 			} );
 		} );
 
 		describe( 'focus behavior', () => {
 			it( 'should focus the popover by default when opened', async () => {
-				await renderAsync(
+				render(
 					<Popover data-testid="popover-element">
 						Popover content
 					</Popover>
 				);
 
-				expect( screen.getByTestId( 'popover-element' ) ).toHaveFocus();
+				const popover = screen.getByTestId( 'popover-element' );
+
+				await waitFor( () => expect( popover ).toBeVisible() );
+				await waitFor( () => expect( popover ).toHaveFocus() );
```

Comment:
Why is the double `await` necessary? Should we add an inline comment with the explanation?

Reply:
It's not necessary actually. I've cleaned the second one up.